src/tfont/converters/ufoConverter.py

```diff
@@ -70,17 +73,18 @@ def open(self, path, font=None):
             master.name = info.styleName
         for blues in (info.postscriptBlueValues, info.postscriptOtherBlues):
             for yMin, yMax in zip(blues[::2], blues[1::2]):
-                master.alignmentZones.append(AlignmentZone(yMin, yMax-yMin))
+                master.alignmentZones.append(AlignmentZone(yMin, yMax - yMin))
         if info.postscriptStemSnapH:
             master.hStems = info.postscriptStemSnapH
         if info.postscriptStemSnapV:
             master.vStems = info.postscriptStemSnapV
-        for g in ufo.guidelines:
-            guideline = Guideline(
-                x=g.x or 0, y=g.y or 0, angle=g.angle or 0, name=g.name or ""
-            )
-            # ufo guideline color and identifier are skipped
-            master.guidelines.append(guideline)
+        if ufo.guidelines:
```

Comment:
This can now be removed.

Reply:
Then we'd have to assume ufoLib2 master?

Reply:
Right, can remove later then.